openstef_dbc/services/write.py

```diff
@@ -178,16 +179,17 @@ def _write_t_ahead_series(
 
         t_adf = forecast.copy()
 
+        # Calculate tAheads
+        timediffs = (
+            t_adf.index.tz_localize(None) - datetime.utcnow().total_seconds() / 3600
+        )
+        # Round it to the first bigger desired_t_ahead
+        t_adf["tAhead"] = round_time_differences(timediffs)
```

Comment:
The `round_time_differences` expects another keyword argument, namely the argument `options`.

Reply:
Yes, fixed!